src/main/java/mate/academy/rickandmorty/model/RickAndMortyModel.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.rickandmorty.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Data;
+
+@Entity(name = "characters")
+@Data
+public class RickAndMortyModel {
```

Comment:
naming doesn't represent what model is for
```suggestion
public class Character {
```

Reply:
+